src/components/SpruceForm/SpruceForm.stories.tsx

```diff
@@ -81,13 +68,13 @@ const example1Def = {
     properties: {
```

Comment:
For every `title` below this line, did you mean to add `Components/`? These are the titles within the form and so they look a little odd

<img width="491" alt="Screen Shot 2022-06-29 at 1 17 40 PM" src="https://user-images.githubusercontent.com/47064971/176497273-c471e634-8517-4bcb-943c-6bb254c780bc.png">

Reply:
You have witnessed the limits of my find and replace skills.